### PR TITLE
Feat/deliverable1

### PR DESCRIPTION
**Pull request by milenasuarezl**
You can see the change in https://milenasuarez.com/2022React-Bootcamp-CapstoneProject/

### CODE REVIEW

File: src/components/Slider/Slider.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Swiper, SwiperSlide } from "swiper/react";
```
Comment (santiagomelowizeline) [inline src/components/Slider/Slider.js:1]: Although is not necessary, try to not use libraries in the project. For instance here you could make a swiper manually or alternatively see how this library works under the hood and try to replicate it

File: src/components/Header/Header.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import Logo from "../Logo/Logo";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
```
Comment (santiagomelowizeline) [inline src/components/Header/Header.js:2]: Try to use images instead of libraries for icons in this project

**Review by santiagomelowizeline — APPROVED** [review]

File: src/components/Slider/Slider.js (hunk 1 above)
Comment (milenasuarezl) [reply src/components/Slider/Slider.js:1]: Thanks for the comment!

File: src/components/Header/Header.js (hunk 2 above)
Comment (milenasuarezl) [reply src/components/Header/Header.js:2]: Thanks for the comment!

**Pull request merged**